src/xunit.execution/Sdk/Frameworks/TestFramework.cs

```diff
@@ -45,7 +45,6 @@ public async void Dispose()
             // of things that will be disconnected.
             await Task.Delay(1);
 
-            ExtensibilityPointFactory.Dispose();
```

Comment:
I think @bradwilson was suggesting moving the `Dispose` method entirely, along with the disposal tracker and instance collection, but I might be mistaken.  I think if we remove this without removing the collection it could result in the instance collection growing unbounded?

Reply:
this would be a bigger change. I thought we wanted to do the minimal amount of work to get unblocked